src/zenml/new/pipelines/pipeline.py

```diff
@@ -637,7 +638,7 @@ def _run(
             stack = Client().active_stack
             stack.validate()
 
-            new_version_requests = self.get_new_version_requests(deployment)
+            self.get_new_version_requests(deployment)
```

Comment:
Seems a little misleading that this is called `get_...` and we don't use the return value

Reply:
Agree, [6f069b6](https://github.com/zenml-io/zenml/pull/2044/commits/6f069b6d768374762d2fa20549805565bc56d8b0)